sql/core/src/test/resources/sql-tests/results/group-by-filter.sql.out

```diff
@@ -0,0 +1,515 @@
+-- Automatically generated by SQLQueryTestSuite
+-- Number of queries: 44
```

Comment:
I've checked that the output except for unsupported ones are the same with pg output: https://gist.github.com/maropu/cfa1343c78277ceeb88eaf5e9e51c3bc

Reply:
I can't visit this link.

Reply:
Really? It seems that link works well.

Reply:
Yes. It's really.